src/widget/city_building_ghost.c

```diff
@@ -351,6 +399,12 @@ static void draw_draggable_reservoir(const map_tile *tile, int x, int y)
             draw_flat_tile(x + X_VIEW_OFFSETS[i], y + Y_VIEW_OFFSETS[i], COLOR_MASK_RED);
         }
     } else {
+        if(!building_construction_in_progress() || draw_later) {
+            if(draw_later) {
```

Comment:
Space after `if` please.
```suggestion
        if (!building_construction_in_progress() || draw_later) {
            if (draw_later) {
```